src/ui/screens/Democracy/DemocracyScreen.tsx

```diff
@@ -244,7 +247,7 @@ function Row({label, children}: {label: string; children: React.ReactNode}) {
 
 const styles = StyleSheet.create({
   listContainer: {
-    padding: standardPadding * 2,
+    paddingVertical: standardPadding * 2,
```

Comment:
Seems like you are just trying to reduce the horizontal padding. That can also be done like following.

```suggestion
    paddingVertical: standardPadding * 2,
    paddingHorizontal: standardPadding,
```

Reply:
The reason I'm doing something like this is that I get a warning as :
<img width="434" alt="image" src="https://user-images.githubusercontent.com/22363352/182841380-9740f463-201b-45ea-9cc4-a59938381c8f.png">

Reply:
I see, what you mean.